packages/ERTP/src/types.js

```diff
@@ -104,7 +104,7 @@
  * represents the issuer they intended, since the same brand can be reused by
  * a misbehaving issuer.
  *
- * @property {(allegedIssuer: any) => boolean} isMyIssuer Should be used with
+ * @property {(allegedIssuer: any) => Promise<boolean>} isMyIssuer Should be used with
```

Comment:
Since the main purpose of our types is as an aid in the IDE to what one is supposed to write at the call site, I don't think `any` is helpful here. I would prefer `allegedIssuer: ERef<Issuer>` here.

Attn @michaelfig I've commented on this before but I don't remember what the response is.

Reply:
I'm in favour.

Reply:
Done!